extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisUtils.scala

```diff
@@ -59,9 +61,10 @@ object KinesisUtils {
       endpointUrl: String,
       checkpointInterval: Duration,
       initialPositionInStream: InitialPositionInStream,
-      storageLevel: StorageLevel): ReceiverInputDStream[Array[Byte]] = {
+      storageLevel: StorageLevel,
+      credentials: Option[AWSCredentials]): ReceiverInputDStream[Array[Byte]] = {
```

Comment:
any thought to using a default param where new DefaultAWSCredentialsProvider() is the default to reduce the API changes?